lib/ecto/changeset.ex

```diff
@@ -2340,10 +2340,13 @@ defmodule Ecto.Changeset do
   `:errors` field of the changeset and the `:valid?` flag will be set to
   `false`.
 
+  Optionally, the validator function can take a third argument with the
+  previous value of the field.
```

Comment:
What do you think about changing the first paragraph to something like this instead? We could probably do a better job describing the old behaviour as well:

```
Validates the given `field` change.

It invokes the `validator` function to perform the validation
only if a change for the given `field` exists and the change
value is not `nil`. If `validator` has an arity of 2, it is passed
`field` as the first argument and the change value as the
second argument. If it has an arity of 3, it is passed a third
argument corresponding to the current value of `field`. The
function must return a list of errors (with an empty list meaning no errors).
```

Reply:
My intention was to modify as little as possible, but it is much better this way. Thanks